plugins/generator-1.20.4/neoforge-1.20.4/templates/elementinits/items.java.ftl

```diff
@@ -103,9 +105,26 @@ public class ${JavaModName}Items {
 	}
 	</#if>
 
+	<#if itemsWithInventory?size != 0>
+		<#list itemsWithInventory as item>
+			public static final DeferredHolder<AttachmentType<?>, AttachmentType<${item.getModElement().getName()}InventoryAttachment>> ${item.getModElement().getRegistryNameUpper()}_INVENTORY =

```

Comment:
What is a proper location to declare these constants and method, given they are generated one after another? Is this version good or move it before block registration helpers?

Reply:
I would put them before methods

Reply:
:heavy_check_mark: Fixed